docs/spfx/publish-to-marketplace-after-publishing.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Maintain and support your published app
+description: Tips to maintain and support your SharePoint Framework app after publishing it to the marketplace
+ms.date: 06/25/2020
+ms.prod: sharepoint
+localization_priority: Priority
+---
+
+# Maintain and support your published app
+
+## Install applications from the marketplace in the tenant app catalog
```

Comment:
Issue: don't start a doc with two headings.

- Docs should have an overview statement between the title & the first subheading